AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java

```diff
@@ -201,7 +201,13 @@
     private RecyclerView mRecyclerView;
     private LinearLayoutManager mRecyclerViewLayoutManager;
     private DeckAdapter mDeckListAdapter;
-    private FloatingActionsMenu mActionsMenu;
+	
+    // private FloatingActionsMenu mActionsMenu;
+	 private FloatingActionButton addDeckButton,addNoteButton,addSharedButton,fabMain;
```

Comment:
The spacing here is strange? It appears you may be using tabs vs spaces

```suggestion
    private FloatingActionButton addDeckButton,addNoteButton,addSharedButton,fabMain;
```

Please go through the entire diff and make sure you are using spaces as specified https://github.com/ankidroid/Anki-Android/wiki/Code-style#white-spaces

Reply:
Also we do not declare multiple items on one line, each declaration gets its own line